features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -7,33 +7,25 @@ import {
 async function relevantZipEntries(zipFile) {
     const entries = await zipFile.getEntries();
     return entries.filter(
-        (each) => !each.includes(".DS_Store") && !each.includes("__MACOSX")
+        (each) =>
+            !each._id.includes(".DS_Store") && !each._id.includes("__MACOSX")
     );
 }
 
 async function readJSONFile(relativeFileName, zipFile) {
-    let fullEntryName;
-    // TODO: This is a hack to handle the fact that ids are
-    // different between the browser pod and the app one.
-    if (zipFile.id.toLowerCase().startsWith("polypod://")) {
-        fullEntryName = `${zipFile.id}/${relativeFileName}`;
-    } else {
-        fullEntryName = `FeatureFiles/${zipFile.id}/${relativeFileName}`;
+    if (!(await zipFile.hasFilePath(relativeFileName))) {
+        throw new MissingFileImportException(relativeFileName);
     }
-
-    return readFullPathJSONFile(fullEntryName, zipFile);
+    const fileEntry = await zipFile.fileEntryForPath(relativeFileName);
+    return readFullPathJSONFile(fileEntry);
 }
 
-async function readFullPathJSONFile(fullEntryName, zipFile) {
-    if (!(await zipFile.hasEntry(fullEntryName))) {
-        throw new MissingFileImportException(fullEntryName);
-    }
-
-    const rawContent = await zipFile.getContent(fullEntryName);
+async function readFullPathJSONFile(fileEntry) {
+    const rawContent = await fileEntry.getContent();
     const fileContent = new TextDecoder("utf-8").decode(rawContent);
 
     if (!fileContent) {
-        throw new MissingContentImportException(fullEntryName);
+        throw new MissingContentImportException(fileEntry);
```

Comment:
Is it possible that this should be fileEntry._id?

Reply:
yes indeed!

Reply:
You could alternatively provide a `toString()` method for `ZipFileEntry` - but I guess it's fine as is for now.